mnd/mnd/urls.py

```diff
@@ -23,8 +23,6 @@
     re_path(r'^carer_operations/?$', CarerOperationsView.as_view(), name='carer_operations'),
     re_path(r'^(?P<registry_code>\w+)/carer_registration/?$', MNDRegistrationView.as_view(), name='carer_registration'),
 
-    re_path(r'^api/v1/', include(('mnd.services.rest.urls.api_urls', 'api_urls'), namespace='v1')),
```

Comment:
This breaks all api urls because of line 13

Reply:
I think just refactor to do urls the same way angelman does

Reply:
Ooops...!
I think I've fixed it now. I tried a couple of the API endpoints and they looked like they were working.

Reply:
p.s. I force pushed the changes.